Docker build failing

```
   Compiling slog-stdlog v4.0.0
error[E0061]: this function takes 4 parameters but 3 parameters were supplied
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/slog-stdlog-4.0.0/lib.rs:244:9
    |
244 |         log::__private_api_log(format_args!("{}", lazy), level, &(target, info.module(), info.file(), info.line()));
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected 4 parameters

error: aborting due to previous error
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -10,7 +10,7 @@ RUN apt-get -q update && \
 RUN \
     cargo --version && \
     rustc --version && \
-    cargo install --path . --root /app
+    cargo install --path . --locked --root /app
 
 FROM debian:buster-slim
 WORKDIR /app

